menuinst/win32.py

```diff
@@ -251,26 +254,28 @@ def remove(self):
 
     def create(self, remove=False):
         args = []
+        # cmd is our root installation interpreter
+        cmd = join(unicode_prefix, u"pythonw.exe").replace("\\", "/")
+        # each of these roll the subprocess cmd into args
         if "pywscript" in self.shortcut:
-            cmd = join(self.menu.prefix, u"pythonw.exe").replace("\\", "/")
+            subprocess_cmd = join(self.menu.prefix, u"pythonw.exe").replace("\\", "/")
             args = self.shortcut["pywscript"].split()
-            args = get_python_args_for_subprocess(self.menu.prefix, args, cmd)
+            args = get_python_args_for_subprocess(self.menu.prefix, args, subprocess_cmd)
         elif "pyscript" in self.shortcut:
-            cmd = join(self.menu.prefix, u"python.exe").replace("\\", "/")
+            subprocess_cmd = join(self.menu.prefix, u"python.exe").replace("\\", "/")
```

Comment:
Ah, I know see that the `"script"` case below uses `u"pythonw.exe"` (even before your changes). Since `"script"` and "`pyscript`" should behave the same concerning terminal windows I'd argue your changes (using `"pythonw"` in any case) should be fine!